crates/p2p/src/client/types.rs

```diff
@@ -268,7 +269,11 @@ impl TryFromDto<p2p_proto::transaction::Transaction> for TransactionVariant {
             }),
             DeployAccountV1(x) => {
                 TransactionVariant::DeployAccountV0V1(DeployAccountTransactionV0V1 {
-                    contract_address: ContractAddress(x.address.0),
+                    contract_address: deployed_contract_address(
```

Comment:
This makes the conversion a blocking call if in async context, so I am wondering if I should move the [parsing](https://github.com/eqlabs/pathfinder/blob/7bd41fc535acad8b8fdcc180c844242fbd819733/crates/p2p/src/client/peer_agnostic.rs#L342) to `tokio::task::spawn_blocking`. Ofc I would have to figure out how to do it only for (deploy accout) transactions. WDYT?

Reply:
It might make sense. I can't really gauge how expensive this might be.. let's wait for someone else to weigh in.

Reply:
A Pedersen hash takes ~22 µs on my computer. The address calculation does 5 + len(constructor_calldata) Pedersen hashes. So I think this usually takes < 0.2 ms, so I'm not sure it's worth it.